src/SignalR/clients/java/signalr/test/build.gradle

```diff
@@ -10,11 +10,10 @@ dependencies {
     implementation 'org.junit.jupiter:junit-jupiter-params:5.11.2'
     testImplementation 'org.junit.jupiter:junit-jupiter:5.11.2'
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.11.2'
-    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'com.google.code.gson:gson:2.8.9'
```

Comment:
Is this bump related to the fix or just for good measure?

Reply:
Shouldn't have been needed, but for some reason when debugging the test in VS Code it complained about a method not being found in gson, command line passed just fine. So I updated the dep and VS Code started working 🤷